infracheck/infracheck/exceptions.py

```diff
@@ -5,6 +5,10 @@ class InfracheckException(Exception):
     pass
 
 
+class CheckNotReadyShouldBeSkippedSignal(InfracheckException):
+    pass
```

Comment:
Is this an exception? (Do I understand this correctly?) If so, should it not be served in any way?

Reply:
What do you mean by served?

Reply:
It is raised there https://github.com/riotkit-org/infracheck/pull/29/files#diff-e58f93086ebe2f552c111b2ae8ebd2f0e974888c200d0bcec992678f32559ab5R61

And handled there: https://github.com/riotkit-org/infracheck/pull/29/files#diff-e58f93086ebe2f552c111b2ae8ebd2f0e974888c200d0bcec992678f32559ab5R121

Used to send a signal to parent context that performing the check should be skipped (reason: already in cache)